src/test/java/eu/europeana/metis/sandbox/controller/DatasetControllerIT.java

```diff
@@ -40,6 +40,9 @@ class DatasetControllerIT {
   @DynamicPropertySource
   public static void dynamicProperties(DynamicPropertyRegistry registry) {
     PostgresContainerInitializerIT.dynamicProperties(registry);
+    PostgresContainerInitializerIT.runScripts(List.of(
```

Comment:
When I did the change for metrics I introduce a new query and the scheduler that reports metrics runs on the background during the tests, and when loading the context this test fails because the tables are not found. So I enabled this.